src/utils/database.ts

```diff
@@ -0,0 +1,40 @@
+import { existsSync, readFileSync, writeFileSync, unlinkSync } from 'fs';
+
+export class Database {
+  defaultFilePath: string;
+
+  constructor() {
+    this.defaultFilePath = 'src/utils/data.json';
+  }
+
+  async hasGameFile(filePathOverride?: string): Promise<boolean> {
+    return existsSync(filePathOverride || this.defaultFilePath);
+  }
+
+  async load(filePathOverride?: string): Promise<object> {
+    try {
+      const data = await readFileSync(
+        filePathOverride || this.defaultFilePath,
```

Comment:
nit: move the right operator of `||` to the default parameter of the function for all functions

Reply:
As an alternative you could move the filename to the constructor